rest/route/commit_queue.go

```diff
@@ -207,7 +208,7 @@ func (cq *commitQueueEnqueueItemHandler) Run(ctx context.Context) gimlet.Respond
 	if patchEmpty {
 		return gimlet.MakeJSONErrorResponder(errors.New("can't enqueue item, patch is empty"))
 	}
-	position, err := cq.sc.EnqueueItem(cq.project, model.APICommitQueueItem{Issue: model.ToStringPtr(cq.item)}, cq.force)
+	position, err := cq.sc.EnqueueItem(cq.project, model.APICommitQueueItem{Issue: model.ToStringPtr(cq.item), Source: restModel.ToStringPtr(commitqueue.SourceCommandLine)}, cq.force)
```

Comment:
how do we know that this is a CLI patch? Is the enqueue handler only used for CLI patches? I see that the delete handler can handle PR ones.

Reply:
I'm still a bit curious about how we know that this one is a diff considering that the delete handler seems to be focused on PRs.

Reply:
I believe PR items are only created when a PR is picked up by the github webhook (in rest/route/github.go). However, PR items are still added to a queue, at which point a user could see it in the queue, and delete it if they choose.

Reply:
Ah that makes sense. Thank you!